addons/website/controllers/main.py

```diff
@@ -660,6 +660,20 @@ def get_new_page_templates(self, **kw):
                 'title': group_el.text,
```

Comment:
We agree that the purpose of this task in to add a new flow to duplicate pages, right ? So that means that if I delete the page that was used as a template, it is okay if the template is deleted too (since there is no page to duplicate anymore) ? And if I edit the page that was used as a template, then the template should be modified too right ? I think yes given what is written in the task but I find it "strange": for me, the page saved as a template should become a real new page template, which would be independent of the real page. 
=> Example: the designer designs a generic page and saves it as a template that you will reuse several times. You will then surely do one of the following:
  - You don't want to touch at what the designer did so you do "new page" every time you want to use it -> this whole page really is just a template and is useless otherwise  
    => better be a real template directly (It also depends if it is more efficient to store a template instead of a page)
  - You finished you website and you clean the useless/unused pages -> you therefore delete the unused template page (if you did not modify it) 
    => you lost the template
  - You do the modifications directly on the page that is used as a template and you save -> you don't like it and want to restart from the template or you want to create a new page with the same template
    => the designer template doesn't exist anymore, as you saved on it 

  If it is okay with PDE then it is okay with me but just telling in case you didn't think of these cases :wink:  In the case where they would be real templates, maybe it could be nice to add a way to delete them (so only in the "Custom" tab) Still in that case, then my first comment could be ignored :slightly_smiling_face:

Reply:
Every reviewer had this kind of remark.
But the feature is not about creating a template from a page, it's only about making a page usable as a template to create other pages. A good practice for the users could be to create unpublished pages that are dedicated to being used as templates, but it is up to them. Modifications on that page being taken into account is a feature: that's the way to make the "template" evolve.
Adding the deletion warning you suggested in your first comment should help to avoid accidental deletions. :+1: